composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=8.2",
-        "php-llm/llm-chain": "0.7.*",
+        "php-llm/llm-chain": "^0.9",
```

Comment:
I think I was wrong with 0.9 here, because if we directly go to `0.9` there will never ever be a bundle which is compatible with `0.8` and as the most hard changes are in `0.8`, I would make 0.8 work here in this PR, release the bundle and then create a PR raising the deposit to 0.9 (which need no changes in the bundle at the moment). WDYT?

Reply:
yea, you're right :)

Reply:
no way around it anymore since i'm not gonna create new 0.8.x patches --- not sure anyone will notice anyways ;)